docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -41,13 +41,15 @@ You can analyze more than one application at a time against more than one transf
 * OpenJDK - upgrades to one of the following JDK versions:
 ** OpenJDK 11
 ** OpenJDK 17
+** OpenJDK 21
 * Linux - ensures there are no Microsoft Windows paths hard-coded into your applications
 * Jakarta EE 9 - for migrating from Java EE 8 to Jakarta EE 9
 * Spring Boot on Red Hat Runtimes
 * Open Liberty
-* Camel - for migrating from Apache Camel 2 to Apache Camel 3
+* Camel - for migrating from Apache Camel 2 to Apache Camel 3, and from Apache Camel 3 to Apache Camel 4
```

Comment:
@emarcusRH 
I don't think the comma is needed here. The phrase cannot be misunderstood without it.